mithril-client/src/type_alias.rs

```diff
@@ -12,6 +12,15 @@ pub use mithril_common::messages::SnapshotMessage as Snapshot;
 ///
 pub use mithril_common::messages::SnapshotListItemMessage as SnapshotListItem;
 
+cfg_unstable! {
+    /// A Cardano node database snapshot
+    ///
+    pub use mithril_common::messages::CardanoDatabaseSnapshotMessage as CardanoDatabaseSnapshot;
+
+    /// List items of Cardano node database snapshot
+    ///
+    pub use mithril_common::messages::CardanoDatabaseSnapshotListItemMessage as CardanoDatabaseSnapshotListItem;
+}
```

Comment:
`ArtifactsLocationsMessagePart` and its three subtypes, `DigestLocation` / `ImmutablesLocation` /`AncillaryLocation` + `MultiFilesUri` / `TemplateUri`, should be exposed too (probably in the `mithril-common re-exports` block).

Reply:
Done